fele-client-service/client-api/scripts/channel.js

```diff
@@ -0,0 +1,28 @@
+const { checkIfNetworkExists, insertToDatabase } = require('../../utils/db')
+const path = require("path");
+const fs = require('fs');
+
+const createChannel = async (networkName, channelName, channelConfigJSON) => {
+    const database = "fele__"+networkName;
+    try{
+        const dbStatus = await checkIfNetworkExists(database)
+        if(dbStatus) {
+            const channelId = await insertToDatabase(database, channelConfigJSON);
+            if(channelId) {
+                var dir = "../../../chaincode/"+networkName+"/"+ channelName
+                dir = path.join(__dirname, dir)
+                if (!fs.existsSync(dir)) {
+                    fs.mkdirSync(dir)
+                }
+                return channelId
+            }
+        }
+    } catch(error) {
+        return false
+    }
+    return false
```

Comment:
If no else part occurs in try then do we need this return ?

Reply:
Not required. we can remove it

Reply:
It is required. if the conditional statements are not met, then a return statement at the end is required